### ISSUE TITLE
Background color is not changed.

### ISSUE THREAD

**Issue by mertnesvat** [LowKostKustomz/StatusAlert#7]
In the example project, I've tried to change background color from apperance but it doesn't work out.

        let statusAlert = StatusAlert()
        statusAlert.appearance.backgroundColor = .red

It's withImage, title and message.

**Comment by LowKostKustomz** [comment]
As you can see in the doc for this property, it is used when blur cannot be used. There is no ability to change background color when blur should be used.

```
/// Used if device does not support blur or if `Reduce Transparency` toggle
/// in `General->Accessibility->Increase Contrast` is on
///
/// - Note: Do not change to save system look
@objc public var backgroundColor: UIColor = UIColor.groupTableViewBackground
```

**Comment by mertnesvat** [comment]
Yes, I tried with reduce transparency is it because of the simulator. I'm not sure 🤔

**Comment by LowKostKustomz** [comment]
Yep, it is bug, thank you! Will fix it as fast as possible and notify you. Stay tuned!

**Comment by mertnesvat** [comment]
Thanks 🙂, actually, maybe instead of changing the background color I need to change blurStyle to the extra light because my friends feedback was the pop up is not readable enough. 

So maybe I would have a feature request that adding blur style to the apperance properties.

**Comment by LowKostKustomz** [comment]
The issue was fixed in version [1.1.0](https://github.com/LowKostKustomz/StatusAlert/releases/tag/1.1.0).

You can also change blur style in appearance. 

There are some source-breaking changes, but it is not too hard to fix them: just rebuild your project, go to `Editor -> Fix All Issues` and here you go.
